### ISSUE TITLE
Build and install from source

### ISSUE THREAD

**Issue by Zed-Wu** [JeanElsner/panda-py#25]
Thanks a lot for your fantastic work! 

I want to modify and fine-tune some C++ code but cannot figure out how to build and install from the source. 
Could you please write a simple guidance of how to build and install the panda_py from the source code? 

update: I successfully build the ``_core.so`` and ``libfranka.so``, and use ``make install`` to install the ``.so`` file, but how to make and install the python library?  Currently I can not ``import panda_py`` from my own project. 

Many thanks.

**Comment by JeanElsner** [comment]
Yes, those shared libraries contain the CPython modules, but they need to additionally be packaged as a proper Python package (there is also some native Python code, e.g. to interact with the Desk). This project uses [scikit-build-core](https://github.com/scikit-build/scikit-build-core) as a build backend, so really the proper way to build from source is to run either `python -m build .` or `pip install . ` to both build and install. I will also provide a proper guide as part of the documentation.
